### ISSUE TITLE
Support dark theme

### ISSUE THREAD

**Issue by marcinsmialek** [joseph-jalbert/colortabs#6]
The add-on should work with dark themes - if I pick dark colors, then the fonts etc. should be brighter. Now it's black text over dark background.
It overrides the default dark theme too. If I switch to a tab with a matching domain, then switch back, then Firefox is bright.

To sum it up
1. Don't override the default theme for non-matching sites  
  Test with the dark theme / other custom themes too
2. Use bright color for items drawn over site-specific dark color background  
  You might use the default theme colors, assuming that person will use similar colors for the sites.

**Comment by jonasflint** [comment]
I agree with this.  You are kind of forced to choose light colors with this addon.

**Comment by hrst** [comment]
I have exactly the two same issues, it is a great addon but these two things would put it to "awesome".

If the dark text color is not possible (maybe all tabs need to use the same, I don't know) then at least being able to use the dark theme as the "default" for all tabs without a specific settings would be crucial.
